src/com/jester/model/Model.java

```diff
@@ -149,9 +153,10 @@ public Music[] view_and_get_MusicRecords() {
                     String fetch_music_ID = rs.getString("music_ID"); ;
                     String fetch_music_title = rs.getString("music_title");
                     String fetch_music_artist = rs.getString("music_artist");
-                    String fetch_music_album = rs.getString("music_artist");
+                    String fetch_music_album = rs.getString("music_album");
                     String fetch_music_path = rs.getString("music_path_to_DIR");
                     
+                    music_as_ls = new Music();

```

Comment:
Declaring a new music object to assign all record attributes and values from the database